vmdb/app/assets/javascripts/cfme_application.js

```diff
@@ -319,8 +319,8 @@ function toggleConvertButtonToLink(button, url, toggle) {
 // parms: button_div=<id of div with buttons to update>, override=<forced state>
 function miqUpdateAllCheckboxes(button_div,override) {
   miqSparkle(true);
-  if ($('masterToggle')) {
-    var state = $('masterToggle').checked;
+  if ($j('#masterToggle')) {
```

Comment:
@h-kataria The jQuery $j('#masterToggle') is not equivalent to the prototype $('masterToggle') is it?  I thought we had to do something with the [0] array element to get back a false if the element id is not present.